keps/sig-node/277-ephemeral-containers/README.md

```diff
@@ -186,16 +186,22 @@ restrictions:
 
 ### Creating Ephemeral Containers
 
-1.  A client fetches the current list of Ephemeral Containers in a pod using
-    `GetEphemeralContainers` in the generated client. This returns a
-    `v1.EphemeralContainers` to which the client appends a new
-    `EphemeralContainer` and then calls `UpdateEphemeralContainers`.
-1.  The apiserver validates and performs the pod update, copying the new 
-    ephemeral container into `Pod.Spec.EphemeralContainers`.
+Ephemeral containers are described in the `EphemeralContainers` field of
+`Pod.Spec`. This must be updated using the `/ephemeralcontainers` subresource,
+similarly to updating `Pod.Status` via `/status`.
+
+The end-to-end process for creating an ephemeral container is:
+
+1.  Fetch a `Pod` object from the `/pods` resource.
+1.  Modify the object and write it back to the pod's `/ephemeralcontainers`
+    subresource, for example using `UpdateEphemeralContainers` in the generated
+    client. (Patching is also supported on `/ephemeralcontainers`.)
+1.  The apiserver validates the update.
```

Comment:
insert a step before this one making it clear that the endpoint only honors data inside spec.ephemeralcontainers

Reply:
Added, thanks.